test/functional/fixtures/multiple-windows/test.js

```diff
@@ -15,287 +15,292 @@ async function assertScreenshotColor (fileName, pixel) {
     expect(assertionHelper.hasPixel(png, pixel, 0, 0)).eql(true);
 }
 
-describe('Multiple windows', () => {
-    describe('Switch to the child window', () => {
-        it('Click on link', () => {
-            return runTests('testcafe-fixtures/switching-to-child/click-on-link.js');
-        });
+if (config.useLocalBrowsers && !config.useHeadlessBrowsers) {
```

Comment:
There is the `!config.useHeadlessBrowsers` condition. However, if we remove the separate `multiple-windows` task we will have only the `test-functional-local-headless-chrome`. Thus, these tests will be ignored